src/assets/translations/en/main.json

```
@@ -406,6 +406,7 @@
         "disabledDepositTitle": "Deposits are temporarily disabled.",
         "haltedWithdrawTitle": "Withdraws are temporarily halted.",
         "disabledWithdrawTitle": "Withdraws are temporarily disabled.",
+        "disabledRunepoolWithdraw": "You have to wait %{days} days to withdraw.",
```

suggestion: Can we give this guy a more declarative name?  e.g:

```suggestion
        "runePoolWithdrawLockedTitle": "You have to wait %{days} days to withdraw.",
```

i.e we already have `disabledWithdrawTitle` and `disabledDepositTitle` copies, which we use when setting he `REACT_APP_FEATURE_SAVERS_VAULTS_DEPOSIT` / `REACT_APP_FEATURE_SAVERS_VAULTS_WITHDRAW`flags (these are useful in case sometimes if off upstream and we need to urgently disable some specific features.

---

q: how will this look like when you are at < 24hrs until maturity, will this show 0 days? Do we want to rather pass the human time using `formatSecondsToDuration`?
Some added benefit of that will be that this will generalize better to other languages while at it!

---

Super legit, changed in [4abc315](https://github.com/shapeshift/web/pull/7472/commits/4abc3154365f8295c06c2fb383b10d762be64282)